curriculum/challenges/english/11-machine-learning-with-python/tensorflow/convolutional-neural-networks.md

```diff
@@ -10,6 +10,12 @@ bilibiliIds:
 dashedName: convolutional-neural-networks
 ---
 
+# --description--
+
+<br>
+
+<small>The next few video lessons will reference [this Colab notebook](https://colab.research.google.com/drive/1ZZXnCjFEOkp_KdNcNabd14yok0BAIuwS#forceEdit=true&sandboxMode=true).</small>
```

Comment:
```suggestion
The next few video lessons will reference [this Colab notebook](https://colab.research.google.com/drive/1ZZXnCjFEOkp_KdNcNabd14yok0BAIuwS#forceEdit=true&sandboxMode=true).
```

We don't really need any of this extra HTML.

Reply:
Sure. I will removed them. Only added them for legibility. Thanks.